cmd/jaeger/integration/datarecivers/jaegerstorage.go

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2024 The Jaeger Authors.
```

Comment:
what is "datareceivers"? If you're modeling it from another PR, then that PR had a reason to do this because of testing, but just to enable Badger for v2 it's not needed - memstore is implemented without any of this complexity

Reply:
removed datarecivers